config/systems.json

```diff
@@ -1,6 +1,6 @@
 {
   "general": {
-    "docker_repository": "spcleth/serverless-benchmarks"
```

Comment:
This is fine of course for testing but it shouldn't be committed :-)

Reply:
Yes, you're right. Fixed it! I accidentally committed all my changes together and forgot to check these files. Thanks for pointing it out!